daffodil-test/src/test/resources/org/apache/daffodil/section15/choice_groups/choice.tdml

```diff
@@ -1838,6 +1838,7 @@ it sure is/
     <tdml:errors>
       <tdml:error>Schema Definition Error</tdml:error>
       <tdml:error>choiceBranchKey</tdml:error>
+      <tdml:error>choiceBranchKeyRanges</tdml:error>
```

Comment:
Nice. Even if you change the message above (the neither-nor change), this test will still be right.